src/common/config_opts.h

```diff
@@ -1294,6 +1294,10 @@ OPTION(rbd_mirror_journal_poll_age, OPT_DOUBLE, 5) // maximum age (in seconds) b
 OPTION(rbd_mirror_journal_max_fetch_bytes, OPT_U32, 32768) // maximum bytes to read from each journal data object per fetch
 OPTION(rbd_mirror_sync_point_update_age, OPT_DOUBLE, 30) // number of seconds between each update of the image sync point object number
 OPTION(rbd_mirror_concurrent_image_syncs, OPT_U32, 5) // maximum number of image syncs in parallel
+OPTION(rbd_mirror_update_replayers_interval, OPT_INT, 30) // interval to update replayers in rbd-mirror daemon
```

Comment:
Longer term planning, the current ```Replayer``` will most likely be renamed to ```PoolReplayer``` and refactored so that it handles all peers for a single conceptual pool. This will allow N-way replication since the ```ImageReplayer``` would need to know all possible remote peer sources for the image so that it can pick the current primary version.